scripts/cli/src/helpers/asset_helper.ts

```diff
@@ -0,0 +1,84 @@
+import type { KeyringPair } from "@polkadot/keyring/types";
+import type { Ticker, Document } from "../types";
+import { sendTx, ApiSingleton } from "../util/init";
+import { assert } from "chai";
+import type { IdentityId } from "../interfaces";
+
+/**
+ * @description Adds Documents for a given token
+ * @param {KeyringPair} signer - KeyringPair
```

Comment:
Specifying types in the comments shouldn't be necessary when using typedoc

Reply:
Also, in general it is recommended to only add documentation for parameters when the name isn't clear enough, and the comment should clarify what the parameter represents. For example

`@param maxSize - number` isn't very helpful
`@param maxSize - max file size in KB. Optional, defaults to 512` would be a better choice

As a rule of thumb, if you can't come up with anything helpful to say about a parameter, it's better not to document it. Typedoc will still show it when generating the documentation

Reply:
Hmm, I understand, makes sense would add these changes as well.